base/doc/ltnews38.tex

```diff
@@ -278,67 +278,149 @@ \section{Hooks, sockets and plugs}
 
 
 
-\section{Extended cross-referencing of document properties}
-
-Traditionally \LaTeX{} allowed with \cs{label} to record two \enquote{local} 
-properties of the document: the representation of the current page number and 
-of the current counter. Their values could then be referenced with the 
-(non-expandable)  commands \cs{ref} and \cs{pageref} in arbitrary places of a 
-document.  With the summer 2023 release the \cs{label} was enhanced to record 
-also if set by packages like \pkg{nameref} or \pkg{hyperref} the title and 
-the name of the link target. 
-
-Over the years packages extended the label-ref system in various ways. 
-Examples are for example the \pkg{refcount} package to reference the values 
-expandably, the \pkg{smartref} package, which allows to store more counter 
-values and so to reference the current chapter together with the current 
-equation, the \pkg{cleveref} package, which stores in a second internal label 
-beside other the name of the counter,  the \pkg{hyperref} package whose 
-\cs{autoref} command tries to retrieve the name of the counter from the name 
-of the link target, the \pkg{tikzmarks} which records absolute positions on 
-the page, and the \pkg{zref} package which implements a general method to 
-record and reference properties. 
-
-Starting with this release the \LaTeX{} kernel contains now its own 
-extension. It is loosely based on \pkg{zref}. It allows to declare new 
-properties, to record arbitrary combinations of properties. The values are 
-retrieved expandably. 
-
-As an example to setup a new property which records the current chapter 
-number the following declaration could be used: 
+\section{Document properties and cross-referencing}
+
+Traditional \LaTeX{} uses \cs{label} to record the values of
+two \enquote{local}
+properties of the document: the textual representations of the
+%% FMi added back in, be cause it is the "current", see Lamport:
+\emph{current
+page number} and the
+%%FMi from Lamport
+the \emph{current \textup{\cs{ref}} value} set by
+\cs{refstepcounter} declarations~\cite[p.~209]{38:Lamport}.
+(These declarations are issued by
+sectioning commands, by numbered environments, and by \cs{item} in an
+\env{enumerate} or a similar environment.)
+
+These two recorded values can then be accessed and typeset
+(from anywhere in the next run of the document) by use of the
+(non-expandable) commands \cs{ref} and \cs{pageref}
+%%FMi new
+using the \emph{key} that was assigned to the values with the \cs{label} command.
+This supported basic cross-referencing (within a document) using
```

Comment:
Otherwise this sentence doesn't have no predicate (usually a verb).
```suggestion
This supported basic cross-referencing (within a document) uses
```

Reply:
Think you are mistaken in this case.

Reply:
You're right, "supported" is the predicate 😄